andela/ah-backend-iroquois
# chore(authors directory): Integrate Coveralls code coverage service

**What does this PR do?** 
   - It Integrates Coveralls code coverage service.

**Description of Tasks to be completed?**
- Updating a travis file with coverage settings.
- Adding a coveralls badge to the repo. 

**How should this be manually tested?**
- After cloning the repo, cd into the directory. 
- Run ```pip install -r requirements.txt```
- Run ``` coverage run --source=authors/ manage.py test```
- Run ```coverage report -m``` to see the coverage report.

**What are the relevant pivotal tracker stories?**

- [#159951999](https://www.pivotaltracker.com/story/show/159951999)
